cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -700,6 +704,16 @@ void writeApplication(String namespaceId, String appId, String versionId,
     updateApplicationEdit(namespaceId, appId);
   }
 
+  @VisibleForTesting
```

Comment:
Please do not move the function if there is no logic change.

Reply:
This is for checkstyle. All methods annotated with the same annotation must be together.
If we mark a method `@VisibleForTesting` we must move them to a section where rest of the `@VisibleForTesting` methods are.